README.md

```diff
@@ -4,3 +4,16 @@ This repo contains basic information about:
 
 - [Docker images produced by the .NET team](./build-info/docker)
```

Comment:
I was thinking we would only provide links for the repos related to the build-info stored in this repo (e.g. the Docker repos).  Maybe including a link to core-repos as a general landing place for other issue.